cinema_project/cinema_project/settings.py

```diff
@@ -110,11 +110,11 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'EET'
```

Comment:
EEST?

Reply:
Django gives an error when adding EEST, it works with EET